DigitalLearningSolutions.Web/Views/VerifyEmail/_NavMenuItems.cshtml

```diff
@@ -0,0 +1,10 @@
+﻿@using DigitalLearningSolutions.Data.Enums
+@using DigitalLearningSolutions.Web.Extensions
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+
+<li class="nhsuk-header__navigation-item @Html.GetSelectedCssClassIfTabSelected(NavMenuTab.LogOut)">
+  <a class="nhsuk-header__navigation-link" asp-controller="Logout" asp-action="Index">
+    Log out
+    <partial name="_NhsChevronRight" />
+  </a>
+</li>
```

Comment:
Does this actually work? The logout endpoint should now be a POST/. I also don't think that this can ever be a selected tab, so I'm not sure we need the css stuff